scaletest/templates/scaletest-runner/metadata_phase.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+if [ -f /tmp/.scaletest_phase_creating_workspaces ]; then
+	echo "Creating workspaces"
+elif [ -f /tmp/.scaletest_phase_ssh ]; then
+	echo "SSH traffic"
+elif [ -f /tmp/.scaletest_phase_rpty ]; then
+	echo "RPTY traffic"
+elif [ -f /tmp/.scaletest_phase_dashboard ]; then
+	echo "Dashboard traffic"
+elif [ -f /tmp/.scaletest_phase_wait_baseline ]; then
+	echo "Waiting $(</tmp/.scaletest_phase_wait_baseline)m (establishing baseline)"
+else
+	echo "None"
```

Comment:
Should "None" happen or is it more like an error state?

Reply:
It’s not really an error state, more like a “nothing yet” state. It looked weird in the dashboard with just an empty string.